libsql-server/src/main.rs

```diff
@@ -63,8 +64,11 @@ struct Cli {
     /// APIs. The key is either a PKCS#8-encoded Ed25519 public key in PEM, or just plain bytes of
     /// the Ed25519 public key in URL-safe base64.
     ///
+    /// It is possible to provide multiple JWT decoding keys in a single file by concatenating them
+    /// together. All decoding keys will be tried when parsing incoming JWT's.
+    ///
     /// You can also pass the key directly in the env variable SQLD_AUTH_JWT_KEY.
-    #[clap(long, env = "SQLD_AUTH_JWT_KEY_FILE")]
+    #[clap(long, env = "SQLD_AUTH_JWT_KEY_FILE", num_args = 1..)]
```

Comment:
what's that for?

Reply:
Good catch! This was done before implementing support for multi key strings and then forgotten. I'll remove.